.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+<!--
+  Thank you for contributing to rufo!
+  Please remember to update the CHANGELOG with the contents of this PR.
+-->
```

Comment:
For those unfamiliar, the contents of this file show up in the description of any new PRs created through the GH web UI. As its am HTML comment even if its not deleted by the submitter, it doesn't appear in the description.

I've found this a useful way to communicate contribution guidelines in other projects, so I've added it here too.

Reply:
This is a great idea!